.github/CONTRIBUTING.md

```diff
@@ -41,10 +41,17 @@ The official tag is `python-attrs` and helping out in support frees us up to imp
 
 You can (and should) run our test suite using [*tox*].
 However, you’ll probably want a more traditional environment as well.
-We highly recommend to develop using the latest Python release because we try to take advantage of modern features whenever possible.
-Also, running [*pre-commit*] later on will require the latest Python version.
 
-First [fork](https://github.com/python-attrs/attrs/fork) the repository on GitHub.
+First, create a [virtual environment](https://virtualenv.pypa.io/) so you don't break your system-wide Python installation.
+We recommend using the Python version from the `.python-version` file in project's root directory.
```

Comment:
@hynek to prevent accidental commits from pyenv users doing `pyenv local`, we'd need to instruct all the contributors to run `git update-index --assume-unchanged .python-version` or a similar exclusion. That is, if you still want to keep this after my other comments.